tracer/transport.go

```diff
@@ -22,6 +22,12 @@ type Transport interface {
 	SetHeader(key, value string)
 }
 
+// NewHTTPTransport returns a new Transport implementation using the given
+// delivery url.
+func NewHTTPTransport(url string) Transport {
```

Comment:
I would second the option to provide hostname and port. This is more flexible and allows users to choose to use environment variables if they like. This doesn't seem to expose any more of the transport mechanism than otherwise allowing configuration via environment variables.